circle.yml

```diff
@@ -0,0 +1,208 @@
+version: 2
+
+workflows:
+  version: 2
+  default:
+    jobs:
+      - prepare:
+          filters:
+            tags:
+              only: /.*/
+      - lint:
+          requires:
+            - prepare
+          filters:
+            tags:
+              only: /.*/
+      - build:
+          requires:
+            - prepare
+          filters:
+            tags:
+              only: /.*/
+      - test-flow:
+          requires:
+            - prepare
+          filters:
+            tags:
+              only: /.*/
+      - test-unit:
+          requires:
+            - prepare
+          filters:
+            tags:
+              only: /.*/
+      - test-render:
+          requires:
+            - prepare
+          filters:
+            tags:
+              only: /.*/
+      - test-query:
+          requires:
+            - prepare
+          filters:
+            tags:
+              only: /.*/
+      - test-expressions:
+          requires:
+            - prepare
+          filters:
+            tags:
+              only: /.*/
+      - deploy-benchmarks:
+          requires:
+            - lint
+            - build
+            - test-flow
+            - test-unit
+            - test-render
+            - test-query
+            - test-expressions
+          filters:
+            tags:
+              only: /v[0-9]+.[0-9]+.[0-9]+(-.+)?/
+            branches:
+              only:
+                - master
+                - /release-.*/
```

Comment:
Why do we need this on `release-` branches if we already have it for version tags?

Reply:
So that when you're benchmarking local changes that will be targeted to a release branch, you can compare `HEAD` to the current state of the target branch.